fox_news_scraper/clean_fn.py

```diff
@@ -0,0 +1,32 @@
+"""A file for cleaning article data by removing stopwords, unwanted
+characters, whitespace etc."""
+
+import re
+
+import nltk
+from nltk.corpus import stopwords
+
+nltk.download("stopwords")
+STOPWORDS = set(stopwords.words("english"))
```

Comment:
Would this global variable work if using the function in another script? It might do I don't know! <3

Reply:
I looked it up and it will! But thanks for flagging this, I wasn't too sure myself until I researched it :)